### PR TITLE
Citus fixes

### PR DESCRIPTION
**Pull request by tanderegg**

### CODE REVIEW

**Comment by tanderegg** [comment]
@rioh Can you review?  Thanks!

File: deploy/roles/postgresql-server-config/tasks/main.yml [hunk 1]
```diff
@@ -16,13 +16,16 @@
   become_method: sudo
   lineinfile:
     dest: "{{ pp_datadir }}/pg_hba.conf"
-    regexp: '^(.*)::1/128(.*)md5'
+    regexp: '^host(.*)postgres(.*)md5'
     backrefs: yes
-    line: '\1::1/128\2trust'
+    line: 'host\1postgres\2trust'
```
Comment (rioh) [inline deploy/roles/postgresql-server-config/tasks/main.yml:21]: Will these work locally?

File: deploy/roles/shared/handlers/main.yml [hunk 2]
```diff
@@ -31,13 +31,17 @@
...
-- name: restart postgres
-  become: yes
-  service:
-    name: "{{ pp_servicename }}"
-    state: restarted
+#- name: restart postgres
+#  become: yes
+#  service:
+#    name: "{{ pp_servicename }}"
+#    state: restarted
```
Comment (rioh) [inline deploy/roles/shared/handlers/main.yml:38]: Would this work if we were to deploy postgres without postgres-config (--tags=postgres) or that wouldn't happen?

File: deploy/roles/postgresql-server-config/tasks/main.yml (hunk 1 above)
Comment (tanderegg) [reply deploy/roles/postgresql-server-config/tasks/main.yml:21]: Added a line to the defaults for pg_hba.conf to make this work in Vagrant.

File: deploy/roles/shared/handlers/main.yml (hunk 2 above)
Comment (tanderegg) [reply deploy/roles/shared/handlers/main.yml:38]: Handlers should run even though they don't have the tag, as long as a task that does have the tag is marked "changed"

**Pull request merged**